source/gui.cpp

```diff
@@ -735,22 +735,38 @@ void reshade::runtime::draw_overlay_menu_home()
 
 		ImGui::SameLine();
 
-		if (ImGui::Button("Active to top", ImVec2(130 - _imgui_context->Style.ItemSpacing.x, 0)))
+		if (ImGui::Button("Sort by used", ImVec2(130 - _imgui_context->Style.ItemSpacing.x, 0)))
```

Comment:
This is nice. But I'd keep the "Active to top" button description, even though it does more, simply for consistency with previous versions.